iwave/optimise.py

```diff
@@ -113,6 +124,10 @@ def cost_function_velocity_depth(
     gauss_width: float
         width of the synthetic spectrum smoothing kernel
 
+    regularisation_param: float
```

Comment:
Some comments/ideas for this parameter. For a user it is not clear what a reasonable value is for `regularisation_param`. Also can we give this a default, `regularisation_param=1` (as done in the downstream functions). Finally, the name of the variable does not explain itself very well. Can we come up with another name that is more intuitive? I´d have to understand better what this penalization does to help with that.